### DISCUSSION TITLE
Support for working with multiple patient species for veterinary antibiograms?

### DISCUSSION THREAD

**Discussion by samueljpearce** [msberends/AMR#130] [Q&A]
Hi, 

Firstly, great work with this package, it's been a great use to try and create some cumulative antibiograms for our data. 

This issue is probably unique to me - I trying to create cumulative antibiograms with veterinary data and have run into a few minor issues that I can work around, but I thought may be worth including in the package for ease.

- I need a variable stating patient species, the only issue is that any column vaguely related to the word "species" gets cleaned by as.mo. I've been working around it by labelling it something completely unrelated but sometimes this does get confusing! Could we exclude a term such as patient_species or similar specifically for this use?
- Veterinary breakpoints are pretty limited and therefore we have to take breakpoints from other places (e.g. human breakpoints) but as.sir does not account for this. Currently, I'm separating the data that we have breakpoints for and analysing those with animal breakpoints separately, and then using as.sir with human breakpoints for the rest, before rejoining the data for antibiogram creation. This works fine, but It would be amazing if as.sir could take species into account patient species and then use the most appropriate breakpoint according to CLSI guidelines.

Would be happy to help contribute towards any of these issues!

**Comment by msberends** [comment]
Hi Sam,

I think I can help you out! Currently busy this week, but you’re on my list to respond soon after!
